.cirrus/Dockerfile

```diff
@@ -15,8 +15,12 @@ RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2
     ./aws/install && \
     rm awscliv2.zip
 
+# Node.js for JS/TS/CSS analysis
+# "build-essential" (containing gcc/g++/make) for C/C++ analysis
+# Webkit libraries for rendering the SonarQube Rule Description view
 ARG NODE_VERSION=20
-RUN apt-get update && apt-get install -y metacity xvfb ffmpeg nodejs=${NODE_VERSION}.* gettext-base \
+RUN apt-get update && apt-get install -y build-essential metacity xvfb ffmpeg nodejs=${NODE_VERSION}.* gettext-base \
```

Comment:
Nitpick: it's a recommended practice to put each package on a separate line, with `\` characters, to make it easier to check the diff when packages are added/removed 🙈